build.gradle

```diff
@@ -80,7 +80,8 @@ pmd {
 }
 
 jacocoTestReport {
-  executionData(test, integration)
+  // TODO: add 'integration' when integration tests are added
+  executionData(test)
```

Comment:
the integration task is executed in travis build so it is no harm in leaving execution data as it was

Reply:
That task is included, but even if it wasn't, without that bit may cause `jacocoTestReport` issues locally until integration tests are created.